internal/pkg/scaffold/helm/dockerfilehybrid.go

```diff
@@ -60,6 +60,13 @@ RUN  /usr/local/bin/user_setup
 {{- if .HelmCharts }}
 COPY helm-charts/ ${HOME}/helm-charts/{{ end }}
 
+{{ if or .HelmCharts .Watches }}
+# Corrects file permissions to be fully executable and writable
+RUN find ${HOME} -type f -exec chmod -R g+rw {} \; && \
+    find ${HOME} -type d -exec chmod -R g+rwx {} \; && \
+	chmod -R g+rwx /usr/local/bin
```

Comment:
Since we _always_ COPY files into `/usr/local/bin` on line 57, this `chmod` command for `/usr/local/bin` should not be part of a conditional. I think we should move it into the existing run command in line 58.

Ditto in the ansible hybrid Dockerfile.